src/implementations/twig/components/footer-core/footer-core.story.js

```
@@ -27,7 +29,12 @@ const prepareData = (data) => {
 export default {
```

Any control to show or hide the logo? 

Here is ticket description:
_"EU footers already have the logo. For EC, it must appear by default, but there should be an option to hide it."_

Same for standardised and harmonised, EU also?

---

I've added a control to hide the logo in harmonised and standardised, not in core where no other "section" can be hidden.